app/misc/authenticate.ts

```diff
@@ -18,15 +19,14 @@ let client;
 let avaterImg;
 let repoList = {};
 let url;
+let account;
 var repoNotFound = 0;
 var signed = 0;
 var changes = 0;
 let signedAfter = false;
-let loginScopes = [
-  "repo",
-  "user"
-];
+let loginScopes = "user repo";
 
+// Configuration of Oauth Application Variables
 const OauthConfig = {
   clientId: 'CLIENT_ID',
```

Comment:
Can we make this env variables? What happens if they are not set?

Reply:
Please let us know what the plan going forward is for the client ID / secret. (Creating a new issue and linking it as a TODO is perfectly acceptable :).

Reply:
This is an issue I have discussed with James, he has agreed to address this after the demo tomorrow.